docker/README.md

```diff
@@ -32,23 +32,7 @@ For all env options see [Config template file](./config-template.yaml) or [Confi
 
 ## Serve UI under a sub-path
 
-To change the public path under which the UI is served you will want to build your own docker image
-
-Clone the repo
-
-``` bash
-git clone https://github.com/temporalio/ui-server.git
-cd ui-server
-```
-
-Build the image
-
-``` bash
-make install-submodules
-docker build -t temporal-ui .
-```
-
-Run the image
+To change the public path under which the UI is served you can use the TEMPORAL_UI_PUBLIC_PATH environment variable.
```

Comment:
this will be overridden by https://github.com/temporalio/ui/blob/main/server/docker/README.md

Reply:
hah, true. Created: https://github.com/temporalio/ui/pull/1556